assets/maps/prefabs/prefab_water_blindpig.dmm

```diff
@@ -1,107 +1,107 @@
-"bu" = (/obj/table/reinforced/bar/auto,/obj/item/storage/box/fruit_wedges{pixel_x = -3},/obj/item/storage/box/cocktail_umbrellas,/obj/item/storage/box/cocktail_doodads{pixel_x = 3},/turf/simulated/floor/black,/area/blind_pig)
-"bv" = (/obj/table/reinforced/bar/auto,/obj/item/reagent_containers/food/drinks/bottle/tequila{pixel_x = 4},/obj/item/reagent_containers/food/drinks/bottle/gin{pixel_x = -4},/turf/simulated/floor/black,/area/blind_pig)
-"bw" = (/obj/storage/crate/freezer,/obj/item/raw_material/ice,/obj/item/raw_material/ice,/obj/item/raw_material/ice,/obj/item/raw_material/ice,/obj/item/raw_material/ice,/obj/item/reagent_containers/food/drinks/bottle/vodka,/turf/simulated/floor/black,/area/blind_pig)
-"bx" = (/obj/item/storage/secure/ssafe/theblindpig{pixel_x = -32},/turf/simulated/floor/plating/random,/area/blind_pig)
-"by" = (/obj/stool/chair/wooden{icon_state = "chair_wooden"; dir = 4},/turf/simulated/floor/carpet/blue/fancy/edge/nw,/area/blind_pig)
-"bz" = (/obj/table/reinforced/bar/auto,/obj/item/reagent_containers/food/drinks/drinkingglass/shot{pixel_x = -6},/turf/simulated/floor/carpet/blue/fancy/edge/north,/area/blind_pig)
-"bA" = (/turf/simulated/floor/carpet/blue/fancy/edge/ne,/area/blind_pig)
-"bB" = (/obj/landmark{name = "waiter"},/turf/simulated/floor/plating/random,/area/blind_pig)
-"bC" = (/obj/shrub{icon_state = "shrub"; dir = 1},/turf/simulated/floor/wood,/area/blind_pig)
-"bD" = (/turf/simulated/floor/carpet/blue/fancy/edge/sw,/area/blind_pig)
-"bE" = (/obj/table/reinforced/bar/auto,/obj/item/reagent_containers/food/drinks/bottle/thegoodstuff{pixel_y = 8},/obj/item/reagent_containers/food/drinks/drinkingglass/shot{pixel_x = 6},/obj/machinery/light/incandescent/warm/very,/turf/simulated/floor/carpet/blue/fancy/edge/south,/area/blind_pig)
-"bF" = (/obj/stool/chair/wooden{dir = 8},/turf/simulated/floor/carpet/blue/fancy/edge/se,/area/blind_pig)
-
-(1,1,1) = {"
-ababababababababababababababababababababababab
-ababababababababaaaaaaaaaaaaaaabababababababab
-ababababaaaaaaaaaaacadaeafagaaaaaaaaababababab
-ababababaaahaiahaaajakalamamaaanaoaaababababab
-ababababaaafamamaaamafamamamaaapaqaaababababab
-ababababaaaaaaaraaaaaaaaaraaaaaaaqaaababababab
-abaaaaaaaaasamamamatauavamamaxaaayaaaaaaababab
-abaaazaAaaafaBaCaCaCaCaCaCaDaEaaaFaGaHaaababab
-abaaamamaIavaJaKaLaKaLaKaMaNamaaaOaFaFaaaaaaaw
-abaaafamaPavaJaQaRaSaTaQaUaNamaVaFaFaFaWaXaYaw
-abaaamamaZavaJaKaMbaaMaKaLaNambbaFaFaFaWaXaYaw
-abaaamamaIavaJaQaUaQaUaQbcbdamaaaFaFaOaaaaaaaw
-abaaaraaaaafbebfbgbhbibfbfbjafaabkaFblaaababab
-abaaaFaFaaasamambmbnboamamamamaaaabpaaaaababab
-abaaaFbqaaaaaabraaaaaaaabraaaaaabsbtaaabababab
-abaaaFbubvbwaFaFaaamafamamamaabxbsbsaaabababab
-abaaaFaFaFaOaFaFaaamambybzbAaabsbBbsaaabababab
-abaaaaaaaaaaaaaaaabCambDbEbFaaaaaaaaaaabababab
-ababababababababaaaaaaaaaaaaaaabababababababab
-ababababababababababababababababababababababab
-"}
+"aa" = (/turf/simulated/wall/auto/reinforced/supernorn,/area/blind_pig)
```

Comment:
Unsure why this prefab changed (could be a subtle change in here that I'm missing). If unintended, could you undo?

Reply:
this was already reverted using git checkout. The formatting was wack when I added a change (only adding _b at the end shirts and pants) so